copier/main.py

```diff
@@ -320,7 +332,7 @@ def answers(self) -> AnswersMap:
             questions.append(
                 Question(
                     answers=result,
-                    ask_user=not self.force,
+                    ask_user=not self.force or self.defaults,
```

Comment:
I'm wondering: shouldn't it be `not self.force and not self.defaults`?

```python
>>> force = False
>>> defaults = True
>>> ask = not force or defaults
>>> ask
True
```

Tell me if I'm wrong but we don't want to ask anything if force is false and we have defaults, right?

Reply:
yes, this should be ` or not self.defaults`

Reply:
as we're removing `force` from the API (keeping the cli), this will become `ask = not defaults`